lib/matplotlib/table.py

```diff
@@ -458,6 +458,7 @@ def get_children(self):
 
     def get_window_extent(self, renderer):
         """Return the bounding box of the table in window coords."""
+        self._update_positions(renderer)
```

Comment:
Could you add a quick test for the issue solved by this change?

Reply:
A few tests in test_bbox_tight fail without it (they test tight-bboxing tables).